cd/manager/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ekidd/rust-musl-builder:stable as builder
+
+RUN USER=root cargo new --bin ceramic-cd-manager
```

Comment:
Why a new cargo project? Doesn't this just add directory name, blank cargo toml, and git init?

Reply:
I followed [this](https://blog.logrocket.com/packaging-a-rust-web-service-using-docker/) article, and I do like their reasoning behind doing things this way. We're constructing the most optimized Dockerfile, which becomes important because Rust has *terrible* compile time.